src/web/src/components/Posts/index.tsx

```diff
@@ -19,6 +19,10 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     postsWrapper: {
       width: '100%',
+      paddingTop: '5rem',
+      [theme.breakpoints.down('xs')]: {
+        paddingTop: '2rem',
+      },
```

Comment:
@PedroFonsecaDEV : As I mentioned earlier in the above comment. I would prefer to use the actual padding property of the `Posts` component instead of using the anchor container. Also in mobile I would prefer a smaller top padding to make it better with the eyes.